add a new product(WEM3050T,released on Nov 2023) introduction (#29848)
home-assistant/home-assistant.github.io: source/_integrations/iammeter.markdown

---

[IAMMETER](https://www.iammeter.com/) provides both a bi-directional single-phase energy meter([WEM3080](https://www.iammeter.com/products/single-phase-meter)) and a bi-directional three-phase energy monitor ([WEM3080T](https://www.iammeter.com/products/three-phase-meter)). Both of them can be integrated into Home Assistant.

> [WEM3050T](https://www.iammeter.com/products/3phase-meter-3050t): latest 3phase/split phase meter, released by IAMMETER on Nov 2023.

Reviewer: ```suggestion
The [WEM3050T](https://www.iammeter.com/products/3phase-meter-3050t) is the latest three-phase/split-phase meter, released by IAMMETER on Nov 2023.
```

> WEM3050T provides nearly all of the features of WEM3080T except the cloud service.

Reviewer: ```suggestion
The WEM3050T provides nearly all of the features the WEM3080T provides, with the exception of the cloud service.
```

Reviewer 2: If I'm reading the GitHub UI correctly seems like there's an extra . there?

Reviewer: thank you for the catch

> The WEM3050T provides nearly all of the features the WEM3080T provides, with the exception of the cloud service..

Reviewer: ```suggestion
The WEM3050T provides nearly all of the features the WEM3080T provides, with the exception of the cloud service.
```

> It can be integrated into Home Assistant by [all of the methods](https://community.home-assistant.io/t/four-ways-to-integrate-the-electricity-usage-monitor-iammeter-into-the-home-assistant/485520/) that WEM3080T supports

Reviewer: ```suggestion
The WEM3050T can be integrated into Home Assistant by [all of the methods](https://community.home-assistant.io/t/four-ways-to-integrate-the-electricity-usage-monitor-iammeter-into-the-home-assistant/485520/) that WEM3080T supports.
```

> Because there is no cloud service by default,  it is much cheaper.

Reviewer: ```suggestion
Because there is no cloud service by default,  the WEM3050T is much cheaper than the WEM3080T.
```

> Because there is no cloud service by default,  the WEM3050T is much cheaper than the WEM3080T.

Reviewer: ```suggestion
Because there is no cloud service by default, the WEM3050T is much cheaper than the WEM3080T.
```
